doc-content/enterprise-only/bpmn/bpmn-gateways-ref.adoc

```diff
@@ -1,104 +1,77 @@
-= Gateways 
-Gateways are used to create or synchronize branches in the workflow using a set of conditions, which is called the gating mechanism. Gateways are of two types:
+[id='bpmn-gateways_{context}']
+= BPMN2 gateways in process designer  
+Gateways are used to create or synchronize branches in the workflow using a set of conditions called the gating mechanism. There are two types of gateways:
 
-* Converging, that is merging multiple flows into one flow.
-* Diverging, that is splitting one Flow into multiple flows.
+* Converging, merging multiple flows into one flow
```

Comment:
I'd say "Converging gateways, merging ..." and "Diverging gateways, splitting ..."